src/views/Update/components/UpdatePage/index.js

```diff
@@ -0,0 +1,365 @@
+import {
+  Typography,
+  Grid,
+  Card,
+  CardContent,
+  CardHeader,
+  Button,
+  TextField,
+} from '@material-ui/core/';
+import React, { useState, useMemo, useEffect } from 'react';
+import { requestInfoUpdate, getAllStates, getAllCountries } from 'services/update';
+import styles from '../Update.module.css';
+import GordonLoader from 'components/Loader';
+import SimpleSnackbar from 'components/Snackbar';
+import GordonOffline from 'components/GordonOffline';
+import useNetworkStatus from 'hooks/useNetworkStatus';
+import { ProfileUpdateField, NotAlumni, ContentCard } from '..';
+import GordonDialogBox from 'components/GordonDialogBox';
+import GordonUnauthorized from 'components/GordonUnauthorized';
+import { ConfirmationRow } from '../ConfirmationRow';
+import { ConfirmationWindowHeader } from '../ConfirmationHeader';
+
+/**
+ * Sends an update form to the development office
+ */
```

Comment:
This comment is both unnecessary and untrue. This comment is for the entire component. Firstly, you should probably rename the component for clarity. Secondly, if you do feel that a comment explaining the purpose of the component is necessary, please make it correct:
```suggestion
/**
 * A form for alumni to request an update to their profile information.
 */
```